src/components/navBar/navBar.js

```diff
@@ -127,7 +130,7 @@ function MdNavBar($mdAria, $mdTheming) {
     template:
       '<div class="md-nav-bar">' +
         '<nav role="navigation">' +
-          '<ul class="_md-nav-bar-list" ng-transclude role="listbox" ' +
+          '<ul class="_md-nav-bar-list" ng-transclude role="tablist" ' +
```

Comment:
The docs on line 46 still refer to `listbox` and should be updated to tablist.

Reply:
Similarly, as you noted in a comment, the docs on lines 34-41, need to be updated to point to the applicable tablist specs.